cnext_server/server/settings.json

```diff
@@ -0,0 +1 @@
+{"loggly-event": true}
```

Comment:
do we need this?

Reply:
yes. we need this to store the log status when the user run the 2nd time